### DISCUSSION TITLE
Require confirmation to mark a PR ready for review

### DISCUSSION THREAD

**Discussion by Pennycook** [community/community#10668] [Mobile]
This is similar to #7743, but since it's a different button I thought I'd open a new topic.

Several times, I've managed to click the "Mark ready for review" button by accident while scrolling through a PR.  The button is large and centered, so very easy to click by mistake.  Since marking a draft ready for review assigns reviewers, updates subscriptions and sends e-mails, such a mistake can inconvenience collaborators.  I could only work out how to revert a PR to draft status using the web UI, and there doesn't seem to be a way to unsubscribe reviewers.

I'd really appreciate an "Are you sure?" confirmation for this button.  I've actually done this enough times that I no longer trust myself to use the app. :smile:

**Comment by taki-on** [comment]
Hi @Pennycook! Thank you for your suggestion! I will forward this to the mobile team, so that we can track this in our back log. Thank you again!
